tools/downloader/topologies.py

```diff
@@ -0,0 +1,99 @@
+import os
+import argparse
+
+from pathlib import Path
+
+import common
+import downloader
+import converter
+
+class Topology:     
+    '''
+    {}
+    Model task type: {}
+
+    License: {}
+    '''
+
+    def __init__(self, model_name, *, precision=None, download_dir=None):
+        '''
+        Downloads target model. If the precision doesn't set, loads all the available models.
+        If there is already downloaded model - retrieves the model from cache.
+        Creates an object which stores information about the downloaded model.
+
+        Parameters
+        ----------
+            model_name
+                Target model name.
+            precision
+                Target model precision. By default is None.
+            download_dir
+                Model download directory.
+        '''
+        if not download_dir: 
+            download_dir = Path(Path('.').resolve() / 'models')
+        cache_dir = Path(Path(download_dir) / '.cache')
+
+        parser = argparse.ArgumentParser()
+        args = argparse.Namespace(all=False, list=None, name=model_name, print_all=False)   
+        topology = common.load_models_from_args(parser, args)[0]
+
+        model_dir = download_dir / topology.subdirectory
+
+        flags = ['--name=' + model_name,
+                 '--output_dir=' + str(download_dir).replace('\\', '\\\\'), 
+                 '--cache_dir=' + str(cache_dir).replace('\\', '\\\\')]
+        if precision:
+            flags += ['--precisions=' + precision]
+        
+        downloader.main(flags)
```

Comment:
Why do you need to use the downloader as a library? It's a tool. Why not just run it?

Reply:
Please tell me which directory should contain the ```open_model_zoo.py``` file.